plugin/mutate/source/dextool/plugin/mutate/backend/analyze/package.d

```diff
@@ -1115,17 +1114,54 @@ struct Analyze {
         if (auto localId = file in result.idFile) {
             const fid = FileId(localId.get);
 
+            int sectionStart = -1;
             auto mdata = appender!(LineMetadata[])();
             foreach (t; tstream.getTokens(file).filter!(a => a.kind == CXTokenKind.comment)) {
                 auto m = matchFirst(t.spelling, re_nomut);
-                if (m.whichPattern == 0)
-                    continue;
 
-                () @trusted {
-                    mdata.put(LineMetadata(fid, t.loc.line,
-                            LineAttr(NoMut(m["tag"], m["comment"]))));
-                }();
-                log.tracef("NOMUT found at %s:%s:%s", file, t.loc.line, t.loc.column);
+                if(m.whichPattern != 0){
```

Comment:
use early exit to reduce identation. you can add back the original code right after "m".

```d
                if (m.whichPattern == 0)
                    continue;
```